test/helpers.py

```diff
@@ -4,6 +4,13 @@
 import re
 from openroad import Design
 
+if os.environ.get("BAZEL_TEST") == "1":
+    from openroad import PlaceOptions
+else:
+    import gpl
+
+    PlaceOptions = gpl.PlaceOptions
```

Comment:
@QuantamHD your solution works for bazel but fails in cmake where `import gpl` is needed.  I settled for this ugly compromise but we do need to revisit the problem.